pkg/asset/manifests/content/bootkube/kube-system-secret-etcd-client.go

```diff
@@ -0,0 +1,25 @@
+package bootkube
+
+import (
+	"text/template"
+)
+
+var (
+	// KubeSystemSecretEtcdClient is the constant to represent contents of kube-system-secret-etcd-client.yaml file
+	KubeSystemSecretEtcdClient = template.Must(template.New("kube-system-secret-etcd-client.yaml").Parse(`
+apiVersion: v1
+kind: Secret
+metadata:
+  name: etcd-client
+  namespace: kube-system
+  labels:
```

Comment:
why so many labels ?

Reply:
For comparison, we have some assets that set `tier` and `k8s-app` (like [this][1]).  We also have some that set `kubernetes.io/cluster-service` (like [this][2]).  I'm not clear on if/how any of these labels matter ;).

[1]: https://github.com/openshift/installer/blob/963d1a2dc8c659315957b0db0d8e1b178659f780/pkg/asset/ignition/bootstrap/content/bootkube_temporary.go#L100-L101
[2]: https://github.com/openshift/installer/blob/963d1a2dc8c659315957b0db0d8e1b178659f780/pkg/asset/ignition/bootstrap/content/bootkube_temporary.go#L282

Reply:
I have deleted all labels.